docs/get-started/athletes.md

```diff
@@ -0,0 +1,65 @@
+(athletes)=
```

Comment:
There are no content changes in this pr. It is just a new syntax! I will submit a future pr with text changes that are reviewable!